install/upgrade_to_2.14.php

```diff
@@ -43,8 +43,26 @@ public function upgrade(Migration $migration) {
        $this->addPropertiesToCategories();
        $this->addTargetActorUnicity();
        $this->addTargetContract();
+       $this->addEntityOption();
    }
 
+
+   public function addEntityOption() {
+      global $DB;
+      $table = 'glpi_plugin_formcreator_entityconfigs';
+
+      if (!$DB->fieldExists($table, 'home_page')) {
+         $this->migration->addField($table, 'home_page', 'integer', ['after' => 'tile_design', 'value' => '-2']);
+         $this->migration->addPostQuery("UPDATE `glpi_plugin_formcreator_entityconfigs` SET `home_page`= 1 WHERE `entities_id` = 0");
+      }
+
+      if (!$DB->fieldExists($table, 'tile_design')) {
+         $this->migration->addField($table, 'tile_design', 'integer', ['after' => 'home_page', 'value' => '-2']);
+         $this->migration->addPostQuery("UPDATE `glpi_plugin_formcreator_entityconfigs` SET `tile_design`= '0' WHERE `entities_id` = 0");
+      }
```

Comment:
this column already exists, these lines should be removed

Reply:
done !